src/tilda.c

```diff
@@ -215,6 +215,33 @@ static void load_custom_css_file () {
     g_free (filename);
 }
 
+static void just_force_dbus_toggle_for_wayland(char window_id)
+{
+    char name[sizeof(TILDA_DBUS_ACTIONS_BUS_NAME) + 1] = TILDA_DBUS_ACTIONS_BUS_NAME;
+    name[sizeof(TILDA_DBUS_ACTIONS_BUS_NAME) - 1] = window_id;
+    name[sizeof(TILDA_DBUS_ACTIONS_BUS_NAME)] = 0;
+    char path[sizeof(TILDA_DBUS_ACTIONS_OBJECT_PATH) + 1] = TILDA_DBUS_ACTIONS_OBJECT_PATH;
+    path[sizeof(TILDA_DBUS_ACTIONS_OBJECT_PATH) - 1] = window_id;
+    path[sizeof(TILDA_DBUS_ACTIONS_OBJECT_PATH)] = 0;
+    GDBusConnection *conn = g_bus_get_sync (G_BUS_TYPE_SESSION, NULL, NULL);
+    GError* error = {0};
+    if (conn)
+    {
+        g_dbus_connection_call_sync (conn,
+                    name, path, "com.github.lanoxx.tilda.Actions", "force_toggle",
+                    NULL, NULL, G_DBUS_CALL_FLAGS_NONE,
+                    -1, NULL, &error);
+        g_object_unref(conn);
+        g_print (_("calling name '%s'.\n"), name);
+        g_print (_("calling path '%s'.\n"), path);
```

Comment:
This would get printed everytime the user toggels the terminal and therefore has the potential to spam the log (or the systemd journal). I am not sure if we need to have this, but if we keep it, then it should be off by default and conditional on the DEBUG option by using the `DEBUG_FUNCTION_MESSAGE` macro.

Reply:
changed